apps/acrs/behaviours/declaration.js

```diff
@@ -0,0 +1,48 @@
+const Utilities = require('../../../lib/utilities');
+
+const CheckUnder18InAggregatedValue = (aggregateArray, fieldLabel) => {
+  const aggregatedLength = aggregateArray.length;
+
+  let responseObject = '';
+  for (let i = 0; i < aggregatedLength; i++) {
+    responseObject = aggregateArray[i]
+      .fields
+      .map(v => v)
+      .filter(field => field.field === fieldLabel);
+
+    if (!Utilities.isOver18(responseObject[0].value)) {
+      return true;
+    }
+  }
+  return false;
+};
+module.exports = superclass => class extends superclass {
+  locals(req, res) {
+    const locals = super.locals(req, res);
+
+    const familyMember = req.sessionModel.get('family-member-in-uk').aggregatedValues;
+    // const refferedSiblings = req.sessionModel.get('referred-siblings').aggregatedValues;
+    const refferedChildren = req.sessionModel.get('referred-children').aggregatedValues;
+    const refferedAddFamily = req.sessionModel.get('referred-additional-family').aggregatedValues;
```

Comment:
Need to add a check that each of these values exists in the session, and maybe that `aggregatedValues` exists for them too, otherwise it will throw an error if they didn't refer in one or more of these sections.